content/operate/kubernetes/architecture/_index.md

```diff
@@ -19,30 +19,30 @@ Kubernetes is an excellent orchestration tool, but it was not designed to deal w
 
 ## Operator based deployment
 
 Operator allows Redis to maintain a unified deployment solution across various Kubernetes environments, i.e., RedHat OpenShift, VMware Tanzu (Tanzu Kubernetes Grid, and Tanzu Kubernetes Grid Integrated Edition, formerly known as PKS), Google Kubernetes Engine (GKE), Azure Kubernetes Service (AKS), and vanilla (upstream) Kubernetes. Statefulset and anti-affinity guarantee that each Redis Enterprise node resides on a Pod that is hosted on a different VM or physical server. See this setup shown in the figure below:
 
-{{< image filename="/images/rs/kubernetes-overview-unified-deployment.png" >}}
+{{< image filename="/images/k8s/kubernetes-overview-unified-deployment.png" >}}
 
 ## Network-attached persistent storage {#networkattached-persistent-storage}
 
 Kubernetes and cloud-native environments require that storage volumes be network-attached to the compute instances, to guarantee data durability. Otherwise, if using local storage,  data may be lost in a Pod failure event. See the figure below:
 
-{{< image filename="/images/rs/kubernetes-overview-network-attached-persistent-storage.png" >}}
+{{< image filename="/images/k8s/kubernetes-overview-network-attached-persistent-storage.png" >}}
 
 On the left-hand side (marked #1), Redis Enterprise uses local ephemeral storage for durability. When a Pod fails, Kubernetes launches another Pod as a replacement, but this Pod comes up with empty local ephemeral storage, and the data from the original Pod is now lost.
 
 On the right-hand side of the figure (marked #2), Redis Enterprise uses network-attached storage for data durability. In this case, when a Pod fails, Kubernetes launches another Pod and automatically connects it to the storage device used by the failed Pod. Redis Enterprise then instructs the Redis Enterprise database instance/s running on the newly created node to load the data from the network-attached storage, which guarantees a durable setup.
 
 Redis Enterprise is not only great as an in-memory database but also extremely efficient in the way it uses persistent storage, even when the user chooses to configure Redis Enterprise to write every change to the disk. Compared to a disk-based database that requires multiple interactions (in most cases) with a storage device for every read or write operation, Redis Enterprise uses a single IOPS, in most cases, for a write operation and zero IOPS for a read operation. As a result, significant performance improvements are seen in typical Kubernetes environments, as illustrated in the figures below:
 
-{{< image filename="/images/rs/kubernetes-overview-performance-improvements-read.png" >}}{{< image filename="/images/rs/kubernetes-overview-performance-improvements-write.png" >}}
+{{< image filename="/images/k8s/kubernetes-overview-performance-improvements-read.png" >}}{{< image filename="/images/k8s/kubernetes-overview-performance-improvements-write.png" >}}
```

Comment:
for some reason in the preview, this read image doesn't appear for me. I think it is still missing from the directory.